tests/scripts/check-generated-files.sh

```diff
@@ -1,6 +1,12 @@
-#!/bin/sh
+#! /usr/bin/env sh
```

Comment:
Not a blocker, just surprised: are there really more systems  with `/usr/bin/env` thand with `/bin/sh`?
(Also, I learned that spaces after `!#` are allowed, which I didn't know.)